src/plugins/application_config/opensearch_dashboards.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "applicationConfig",
+  "version": "1.0.0",
```

Comment:
How do we use version if this is a core plugin?

Reply:
Good call out. Let me check other core plugins

Reply:
Updated to be `opensearchDashboards`